generators/entity/prompts.js

```diff
@@ -830,6 +837,7 @@ function askForField(done) {
         if (props.fieldAdd) {
             if (props.fieldIsEnum) {
                 props.fieldType = _.upperFirst(props.fieldType);
+                props.fieldValues = props.fieldValues.toUpperCase();
```

Comment:
This also fixes an issue when you run `export-jdl` but where the fieldValues contain lower-case letters.  JDL exects enum values in all caps.